runtime/parachains/src/hrmp.rs

```diff
@@ -2009,3 +2140,251 @@ mod tests {
 		});
 	}
 }
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking {
```

Comment:
Why not put this in a separate file instead?

Reply:
looking at the whole crate, it seems like the pattern is one file per pallet (main code + test, and now adding benchmarks), but happy to revamp if you think I am wrong.

Reply:
Recent migration for the `configuration` changed this.

Reply:
UPDATE: I see that you've changed this pattern in the `configuration`. Why is there a `configuration` folder and a `configuration.rs`? shouldn't `configuration.rs` be the `mod.rs` of `configuration` folder? I think that's a good pattern, and would like to do it that way for this pallet as well.

Reply:
That's actually a new format in Rust 2018 -- instead of a `configuration/mod.rs` file, you can have a `configuration.rs` file to replace it and all the files under the `configuration/` directory would work as if they're modules of `configuration.rs`.